pkg/shim/scheduler_mock.go

```diff
@@ -21,13 +21,16 @@ import (
 	"github.com/cloudera/yunikorn-core/pkg/api"
 	utils "github.com/cloudera/yunikorn-core/pkg/common/configs"
 	"github.com/cloudera/yunikorn-core/pkg/entrypoint"
+	"github.com/cloudera/yunikorn-core/pkg/log"
 	"github.com/cloudera/yunikorn-k8shim/pkg/cache"
 	"github.com/cloudera/yunikorn-k8shim/pkg/callback"
 	"github.com/cloudera/yunikorn-k8shim/pkg/client"
 	"github.com/cloudera/yunikorn-k8shim/pkg/common"
 	"github.com/cloudera/yunikorn-k8shim/pkg/common/test"
+	utils2 "github.com/cloudera/yunikorn-k8shim/pkg/common/utils"
```

Comment:
Can we rename the utils import (line 22) and change then two references to that and keep this as utils?

Reply:
Done